lib/pwwka/configuration.rb

```diff
@@ -31,6 +32,7 @@ def initialize
       @keep_alive_on_handler_klass_exceptions = false
       @async_job_klass = Pwwka::SendMessageAsyncJob
       @default_prefetch = nil
+      @allow_all_schemas = false
```

Comment:
I like the way you did this, but I think a better name for this behavior might be something like `parse_payload` and set to `true`?  Or maybe `receive_raw_payload` defaulting to `false`?

Reply:
I like `receive_raw_payload` this change is coming up in a bit.